src/lib.rs

```diff
@@ -1511,17 +1511,10 @@ impl ColorSpec {
         &self,
         console: &mut wincolor::Console,
     ) -> io::Result<()> {
-        use wincolor::Intense;
-
-        let intense = if self.intense { Intense::Yes } else { Intense::No };
-
-        let fg_color = self.fg_color.as_ref().and_then(|c| c.to_windows());
-        if let Some(color) = fg_color {
+        if let Some((intense, color)) = self.fg_color.and_then(|c| c.to_windows(self.intense)) {
```

Comment:
Could you please make sure lines are wrapped to 79 columns inclusive? Similarly for the line below.

(This is why intermediate names were used before. Getting rid of them means you blow the line budget.)